shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -1 +1,460 @@
-mod basic_driver_tests;
+mod cache;
+mod collections;
+mod functions;
+mod keyspace;
+mod native_types;
+mod prepared_statements;
+#[cfg(feature = "alpha-transforms")]
+mod protect;
+mod table;
+mod udt;
+
+use crate::helpers::cassandra::{assert_query_result, execute_query, run_query, ResultValue};
+use crate::helpers::ShotoverManager;
+use cassandra_cpp::{stmt, Batch, BatchType, Error, ErrorKind, Session};
+use futures::future::{join_all, try_join_all};
+use serial_test::serial;
+use test_helpers::docker_compose::DockerCompose;
+
+fn test_batch_statements(connection: &Session) {
```

Comment:
Might as well move this too so this file is just tests

Reply:
ah good point